cli/build.rs

```diff
@@ -352,9 +352,18 @@ fn create_cli_snapshot(snapshot_path: PathBuf) -> CreateSnapshotOutput {
   use deno_runtime::deno_cron::local::LocalCronHandler;
   use deno_runtime::deno_http::DefaultHttpPropertyExtractor;
   use deno_runtime::deno_kv::sqlite::SqliteDbHandler;
+  use deno_runtime::ops::bootstrap::SnapshotOptions;
   use deno_runtime::permissions::PermissionsContainer;
   use std::sync::Arc;
 
+  fn deno_version() -> String {
+    if env::var("DENO_CANARY").is_ok() {
+      format!("{}+{}", env!("CARGO_PKG_VERSION"), git_commit_hash())
+    } else {
+      env!("CARGO_PKG_VERSION").to_string()
+    }
+  }
```

Comment:
Isn't this duplicated in `cli/version.rs`? Could we maybe reuse some of the code?

Reply:
env!(DENO_CANARY) macro won't work from build.rs